casepropods/family_connect_registration/plugin.py

```diff
@@ -1,13 +1,51 @@
+from confmodel import fields
+from casepro.cases.models import Case
 from casepro.pods import Pod, PodConfig, PodPlugin
+import requests
 
 
 class RegistrationPodConfig(PodConfig):
-    pass
+    url = fields.ConfigText("URL to query for the registration data",
+                            required=True)
+    token = fields.ConfigText("Authentication token for registration endpoint",
+                              required=True)
+    field_mapping = fields.ConfigList(
+        "Mapping of field names to what should be displayed for them",
```

Comment:
I'm against the custom component because it seems overkill and there isn't really the time to do it now.
I'm not sure how to add an example. Is there a special argument for it to the config component, should I add it to the description or just as a comment? Or is there some other way you'd suggest?